### DISCUSSION TITLE
Internal Implementation of Quartznet Scheduler.

### DISCUSSION THREAD

**Discussion by VedhaGanesh** [quartznet/quartznet#1895] [Q&A]
Hello, I just came across Quartz Scheduler and I would like to know what implementation or logic does quartz use for Scheduling a Job?

**Comment by lahma** [comment]
You can browse the source right here on GitHub. [QuartzSchedulerThread](https://github.com/quartznet/quartznet/blob/main/src/Quartz/Core/QuartzSchedulerThread.cs) is a good starting point.
